src/app/app.module.ts

```diff
@@ -117,6 +118,8 @@ import { InfiniteScrollModule } from 'ngx-infinite-scroll';
 import { CoreModule } from './component/core/core.module';
 import { EcoNewsModule } from './component/eco-news/eco-news.module';
 import { AuthModule } from './component/auth/auth.module';
+import {NewsPreviewPageComponent} from "./component/eco-news/news-preview-page/news-preview-page.component";
```

Comment:
Please format your code and use single quotes

Reply:
fixed